pallets/multisig/src/lib.rs

```diff
@@ -650,6 +663,18 @@ decl_module! {
                 optional_cdd_auth_id
             )
         }
+
+        /// An internal call to execute a scheduled multisig proposal.
+        #[weight = 500_000_000]
+        fn execute_scheduled_proposal(
```

Comment:
Are we adding the benchmarked weight of this extrinsic in `create_proposal_xx()` extrinsic ?

Reply:
Ignore all the weights stuff for now. Benchmarks and weights will be fixed in a different PR.